index.html

```diff
@@ -1,171 +1,287 @@
+          </li>
+          <li>
+            <a href="#"><img src="./images/Tag (2).png" alt="css" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/Tag (3).png" alt="javascript" /></a>
+          </li>
         </ul>
 
-</section>
-<section class="works">
-<div class="cardworks-one">
-<img src="./images/Snapshoot Portfolio.png" alt="" class="snapshoot">
-<div class="left-blockone"></div>
-<h2 class="headings">Tonic</h2>
-<img src="./images/Frame 2 (1).png" alt="" class="frame">
-  <p class="page">
- A daily selection of privately personalized reads; no accounts or sign-ups required.
-   
+        <div class="buttons">
+          <button class="press">See project</button>
+        </div>
+      </div>
+      <div class="cardworks-two">
+        <img
+          src="./images/Snapshoot Portfolio (2).png"
+          alt=""
+          class="snapshoot"
+        />
+        <div class="left-blockone"></div>
+        <h3 class="heading-two">Multi-post stories</h3>
+        <img src="./images/Frame 2 (1).png" alt="" class="frame" />
+        <p class="page">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tag">
+          <li>
+            <a href="#"><img src="./images/Tag (1).png" alt="html" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/Tag (2).png" alt="css" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/Tag (3).png" alt="javascript" /></a>
+          </li>
+        </ul>
 
-  </p>
-  <ul class="tag">
-        <li><a href="#"><img src="./images/Tag (1).png" alt="html"></a></li>
-        <li><a href="#"><img src="./images/Tag (2).png" alt="css"></a>
-        <li><a href="#"><img src="./images/Tag (3).png" alt="javascript"></a>
-         </ul> 
-  
-       
-      <div class="buttons">
-       <button class="press">See project</button> 
+        <div class="buttons">
+          <button class="press">See project</button>
+        </div>
       </div>
-    </div>
-<div class="cardworks-two">
-  <img src="./images/Snapshoot Portfolio (2).png" alt="" class="snapshoot">
-  <div class="left-blockone"></div>
-  
-   <h3 class="heading-two">Multi-post stories</h3> 
-  
-  
-<img src="./images/Frame 2 (1).png" alt="" class="frame">
-  <p class="page">
-    A daily selection of privately personalized reads; no accounts or sign-ups required.
-  </p>
-  
-  <ul class="tag">
-        <li><a href="#"><img src="./images/Tag (1).png" alt="html"></a></li>
-         <li><a href="#"><img src="./images/Tag (2).png" alt="css"></a>
-         <li><a href="#"><img src="./images/Tag (3).png" alt="javascript"></a>
-         </ul> 
-      <div class="buttons">
-       <button class="press">See project</button> 
-      </div> 
+      <div class="cardworks-three">
+        <img
+          src="./images/Snapshoot Portfolio (3).png"
+          alt=""
```

Comment:
[OPTIONAL]

- [ ] I recommend adding proper value for the `alt` attribute. This will help your project become more accessible :rocket:.